core/src/test/scala/unit/kafka/server/MetadataCacheTest.scala

```diff
@@ -483,61 +475,4 @@ class MetadataCacheTest {
     assertEquals(initialBrokerIds.toSet, aliveBrokersFromCache.map(_.id).toSet)
   }
 
-  @Test
-  def testGetClusterMetadataWithOfflineReplicas(): Unit = {
```

Comment:
Why remove this?

Reply:
Hmm, I think this was an artifact of the merge, I'll restore this test